package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "apostrophe-address-widgets",
+  "version": "1.0.0",
+  "description": "Address widgets for ApostropheCMS",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/punkave/apostrophe-address-widgets.git"
+  },
+  "keywords": [
+    "ApostropheCMS",
+    "Address",
+    "Widgets"
+  ],
+  "author": "P'unk Ave",
+  "license": "ISC",
```

Comment:
Our license is MIT. Please change ISC to MIT anywhere you may have inadvertently specified the wrong (but fortunately similar) license on modules.

Reply:
See 5696d2a